src/mjolnir/ferry_connections.cc

```diff
@@ -70,7 +70,7 @@ std::pair<uint32_t, bool> ShortestPath(const uint32_t start_node_idx,
     overall_access_before = overall_access_after;
```

Comment:
Previously only the path with `overall_access_after == 0` has been found for LeShuttle entrance, preventing this algorithm from further iterations